backend/init-mongo.js

```diff
@@ -0,0 +1,16 @@
+db.createUser(
```

Comment:
This file and `docker-compose.yml` do not really belong to backend so I would suggest moving it to better organize the project. What I like to do is create a `docker` directory in the root of the project and put all docker related files there.

Also the formatting is messy (lots of unnecessary new lines) and no new line at the end of the file.

Reply:
Also I do not think you need this file at all, you already create a admin account by using the docker environment variables `MONGO_INITDB_ROOT_USERNAME` and `MONGO_INITDB_ROOT_PASSWORD`. I might be wrong though so double check that.

Reply:
What about multiple `Dockerfile`s (one for backend and one for frontend) in this `docker` directory?
Dockerfile for React is not done yet but I thought it'd be nice to build entire app by simple `docker-compose up`.

Reply:
You can name the `Dockerfile`s however you want. Naming it `Dockerfile` is convenient because it is the default name that docker is looking for when building an image, but you can very easily overwrite this. We for example have an app and have to dockerfiles next to each other named `Dockerfile.frontend` and `Dockerfile.backend`

Reply:
Of course the project structure is up to you, I'm just providing you more solutions in case you did not know about them.